website/content/docs/k8s/operations/gossip-encryption-key-rotation.mdx

```diff
@@ -10,111 +10,133 @@ The following instructions provides a step-by-step manual process for rotating [
+    `Wa6/XFAnYy0f9iqVH2iiG+yore3CqHSemUy4AIVTa/w=` as the replacement gossip encryption key.
 
-  `kubectl exec` into a Consul Agent pod (Server or Client) and add the new key to the Consul Keyring. This can be performed by running the following command:
+1. Add new key to consul keyring.
 
-  **Note:** If ACLs are enabled, export the bootstrap token as the CONSUL_HTTP_TOKEN to perform all `consul keyring` operations. The bootstrap token can be found in the Kubernetes secret `consul-bootstrap-acl-token` of the primary datacenter.
+    1. `kubectl exec` into a Consul Agent pod (Server or Client) and add the new key to the Consul Keyring. This can be performed by running the following command:
 
-  ```shell-session
-  consul keyring -install="Wa6/XFAnYy0f9iqVH2iiG+yore3CqHSemUy4AIVTa/w="
-  ==> Installing new gossip encryption key...
-  ```
-  Consul automatically propagates this encryption key across all Clients and Servers across the cluster and the federation if Consul federation is enabled.
+        ```shell-session
+        kubectl exec -it consul-server-0 -- /bin/sh
+        ```
 
-  List the keys in the keyring to verify the new key has been installed successfully.
+    1. **Note:** If ACLs are enabled, export the bootstrap token as the CONSUL_HTTP_TOKEN to perform all `consul keyring` operations. The bootstrap token can be found in the Kubernetes secret `consul-bootstrap-acl-token` of the primary datacenter.
 
-  ```shell-session
-  consul keyring -list
-  ==> Gathering installed encryption keys...
+        ```shell-session
+        export CONSUL_HTTP_TOKEN=<INSERT BOOTSTRAP ACL TOKEN>
+        ```
+    
+    1. Install the new Gossip encryption key with the `consul keyring` command:
+
+        ```shell-session
+        consul keyring -install="Wa6/XFAnYy0f9iqVH2iiG+yore3CqHSemUy4AIVTa/w="
+        ==> Installing new gossip encryption key...
+        ```
+        Consul automatically propagates this encryption key across all Clients and Servers across the cluster and the federation if Consul federation is enabled.
+
+    1. List the keys in the keyring to verify the new key has been installed successfully.
+
+        ```shell-session
+        consul keyring -list
+        ==> Gathering installed encryption keys...
+
+        WAN:
+          CL6M+jKj3630CZLXI0IRVeyci1jgIAveiZKvdtTybbA= [2/2]
+          Wa6/XFAnYy0f9iqVH2iiG+yore3CqHSemUy4AIVTa/w= [2/2]
+
+        dc1 (LAN):
+          CL6M+jKj3630CZLXI0IRVeyci1jgIAveiZKvdtTybbA= [4/4]
+          Wa6/XFAnYy0f9iqVH2iiG+yore3CqHSemUy4AIVTa/w= [4/4]
+
+        dc2 (LAN):
+          CL6M+jKj3630CZLXI0IRVeyci1jgIAveiZKvdtTybbA= [4/4]
+          Wa6/XFAnYy0f9iqVH2iiG+yore3CqHSemUy4AIVTa/w= [4/4]
+        ```
+
+1. Use the new key as the gossip encryption key.
+
+    1. Once the new key has been added to the keychain, you can install it as the new gossip encryption key. In order to do so
+    run the following while remaining `kubectl exec`'d in the Consul Agent pod.
```

Comment:
```suggestion
    1. After the new key has been added to the keychain, you can install it as the new gossip encryption key. Run the following command in the Consul Agent pod using `kubectl exec`:
```
Would it be better to just add `kubectl exec` to the command? I was just trying to get rid of the awkward "`kubectl exec`'d" construction.